src/brogue/MainMenu.c

```diff
@@ -260,136 +279,336 @@ void initializeMenuFlames(boolean includeTitle,
+        initializeMainMenuButton(&(buttons[1]), "  New %sS%seeded Game  ", 's','S', NG_NEW_GAME_WITH_SEED);
+        initializeMainMenuButton(&(buttons[2]), "     %sL%soad Game     ", 'l','L', NG_OPEN_GAME);
+
+    } else if (rogue.nextGame == NG_FLYOUT_VIEW) {
+
+        buttonCount = 2;
+        initializeMainMenuButton(&(buttons[0]), "   View %sR%secording  ", 'r','R', NG_VIEW_RECORDING);
+        initializeMainMenuButton(&(buttons[1]), "    %sH%sigh Scores    ", 'h','H', NG_HIGH_SCORES);
+
+    } else if (rogue.nextGame == NG_FLYOUT_OPTIONS) {
+
+        buttonCount = 2;
+        initializeMainMenuButton(&(buttons[0]), "    Game V%sa%sriant   ", 'a','A', NG_NOTHING);
+        initializeMainMenuButton(&(buttons[1]), "     Game %sM%sode     ", 'm','M', NG_GAME_MODE);
+
+    } else {
+        return;
+    }
+
+    stackButtons(buttons, buttonCount, position, 2, false);
+    initializeMenu(menu, buttons, buttonCount, shadowBuf);
+}
+
+/// @brief Displays a dialog window for the user to chose a game mode. The game mode is displayed in the bottom left
+/// on the title screen (except normal mode).
+static void chooseGameMode() {
+    short gameMode;
+    char textBuf[COLS * ROWS] = "", tmpBuf[COLS * ROWS] = "", goldColorEscape[5] = "", whiteColorEscape[5] = "";
+
+    encodeMessageColor(goldColorEscape, 0, &yellow);
+    encodeMessageColor(whiteColorEscape, 0, &white);
+
+    sprintf(tmpBuf,"%sNormal Mode%s\n",goldColorEscape,whiteColorEscape);
+    strcat(textBuf, tmpBuf);
+    strcat(textBuf, "Punishingly difficult. Maliciously alluring. Perfectly normal.\n\n");
+
+    sprintf(tmpBuf,"%sEasy Mode%s\n",goldColorEscape,whiteColorEscape);
+    strcat(textBuf, tmpBuf);
+    strcat(textBuf, "Succumb to demonic temptation and play as an all-powerful ampersand, taking 20%% as much damage. But great power comes at a great price -- specifically, a 90% income tax rate.\n\n");
+
+    sprintf(tmpBuf,"%sWizard Mode%s\n",goldColorEscape,whiteColorEscape);
+    strcat(textBuf, tmpBuf);
+    strcat(textBuf, "Play as an invincible wizard that starts with legendary items and is magically reborn after every death. Summon monsters and make them friend or foe. Conjure any item out of thin air. All this and more, for the bargain basement price of forfeiting your score.");
+
+    brogueButton buttons[3];
+    cellDisplayBuffer rbuf[COLS][ROWS];
+    copyDisplayBuffer(rbuf,displayBuffer);
+    initializeMainMenuButton(&(buttons[0]), "      %sW%sizard       ", 'w','W', NG_NOTHING);
+    initializeMainMenuButton(&(buttons[1]), "       %sE%sasy        ", 'e','E', NG_NOTHING);
+    initializeMainMenuButton(&(buttons[2]), "      %sN%sormal       ", 'n','N', NG_NOTHING);
+    gameMode = printTextBox(textBuf, 20, 5, 66, &white, &black, rbuf, buttons, 3);
+    overlayDisplayBuffer(rbuf, NULL);
+    if (gameMode == 0) {
+        rogue.wizard = true;
+        rogue.easyMode = false;
+    } else if (gameMode == 1) {
+        rogue.wizard = false;
+        rogue.easyMode = true;
```

Comment:
See `enableEasyMode()`, also changes `player.info.displayChar = '&';`. Perhaps pull this out as a function now used in 2 places.

Reply:
Ah right. Probably makes sense to use a different character for wizard mode, too, but that can be a future enhancement.

Reply:
Done. I added the game mode to the "View Dungeon Seed" message as well.